test/__snapshots__/storyshots.spec.js.snap

```diff
@@ -9870,10 +9870,10 @@ exports[`Storyshots Config Provider Spacing Aliases 1`] = `
         style={
           Object {
             "backgroundColor": "#efefef",
-            "borderBottomWidth": 48,
```

Comment:
I'm stumped why these values changed. When loading storybook in browser, the styles are correct, and screenshot comparisons are passing. Is this a limitation with how context works within storyshots perhaps?

Reply:
Oh interesting it's only affecting those set by context. It does sound like an enzyme limitation not liking the changes but not sure if there's much we can do about it.

As far as we have this covered with other tests (like the image comparison ones you mentioned) I think we are good